bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -42,23 +42,75 @@ spec:
         - |
           #!/bin/bash
           set -euo pipefail
+          set -x
+          if [[ -f /usr/bin/id ]]; then
```

Comment:
`-x` for executable?  

I don't understand all these checks, they are not atomic and thus worse than redundant because they hid the real error message.

`/usr/bin/id openvswitch || true`

Reply:
or no `|| true` with the `set -e`

Reply:
or I guess it's not clear in which environment `id` is not present?

Reply:
> I don't understand all these checks, they are not atomic and thus worse than redundant because they hid the real error message.

Most (if not all) of these checks were added to debug and will not exist in the final PR.